shamie-Resnet.mdx

```diff
@@ -0,0 +1,20 @@
+Resnet Paper Summary:
+
+Introduction:
+Observed Problem: Degradation Problem
+As the depth of the neural network deepens, the accuracy starts to saturate.
+
+The pre-existing perspectives of deeper layers were always thought of as effective because the perfomance of neural networks improved a lot by deepening the layers,

```

Comment:
```suggestion
The pre-existing perspectives of deeper layers was always thought of as effective because the performance of neural networks improved a lot by deepening the layers.

```